src/static/css/variables.css

```diff
@@ -0,0 +1,18 @@
+:root {
+    /* Colors */
+    --color-blue-100: hsl(234, 100%, 96%);
```

Comment:
As per our earlier discussion, let's make these two variables like so:

```css
    --color-blue-100-hsl: 234, 100%, 96%;
    --color-blue-100: hsl(var(--color-blue-100-hsl);
```

That way we can use the HSL string variable to construct different opacity variants, like this:

```css
    --color-blue-100-50-percent: hsl(var(--color-blue-100-hsl) / 50%);
```